### ISSUE TITLE
Some errors have occurred when executing the build with Go 1.9

### ISSUE THREAD

**Issue by jumpeiMano** [guregu/kami#32]
Hello,

When I executed the build with Go 1.9, some errors have occurred.
```
vendor/github.com/guregu/kami/handler_17.go:47:2: duplicate case func("context".Context, http.ResponseWriter, *http.Request) in type switch
	previous case at vendor/github.com/guregu/kami/handler_17.go:45:2
vendor/github.com/guregu/kami/middleware_17.go:151:2: duplicate case func("context".Context, http.ResponseWriter, *http.Request) "context".Context in type switch
	previous case at vendor/github.com/guregu/kami/middleware_17.go:149:2
vendor/github.com/guregu/kami/middleware_17.go:207:2: duplicate case func("context".Context, mutil.WriterProxy, *http.Request) "context".Context in type switch
	previous case at vendor/github.com/guregu/kami/middleware_17.go:205:2
vendor/github.com/guregu/kami/middleware_17.go:215:2: duplicate case func("context".Context, *http.Request) "context".Context in type switch
	previous case at vendor/github.com/guregu/kami/middleware_17.go:211:2
vendor/github.com/guregu/kami/middleware_17.go:223:2: duplicate case func("context".Context) "context".Context in type switch
	previous case at vendor/github.com/guregu/kami/middleware_17.go:219:2
vendor/github.com/guregu/kami/middleware_17.go:235:2: duplicate case func("context".Context, http.ResponseWriter, *http.Request) "context".Context in type switch
	previous case at vendor/github.com/guregu/kami/middleware_17.go:231:2
```

As an example, it appears to be occurring [here](https://github.com/guregu/kami/blob/master/handler_17.go#L47).
In Go 1.9, is that the treatment of "context" and "netcontext" become equivalent...?

Anyway, I think it may be necessary to add a build tag...

Thanks!

**Comment by guregu** [comment]
Thank you for the report! I have fixed it in fa67a6fd9f9e27b1a3008627fd634c22563af4a8.

**Comment by jumpeiMano** [comment]
Oh, thank you very much for your prompt response!!!!
